setup.py

```diff
@@ -116,14 +116,26 @@ def get_all_files_in_dir(directory, pattern):
 ]
 
 setup(
+    python_requires=">=3.8",
+    classifiers=[
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: BSD 3-Clause License",
+        "Natural Language :: English",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
Apparently, the package can be built on Python 3.10 based on CI/CD. 

I opted not to update our Anaconda environments to 3.10 just yet though.